e2e-tests/conf/ca.crt

```diff
@@ -0,0 +1,10 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Can we use it from TLS test? Why do we need to have almost the same files in different dirs?

Reply:
done